File: crowntest-sdk/src/main/java/com/crowntest/sdk/exceptions/ScreenshotFailedException.java
package com.crowntest.sdk.exceptions;

public class ScreenshotFailedException extends RuntimeException {

    public ScreenshotFailedException(String message) {
        super(message);
    }

    public ScreenshotFailedException(String message, Throwable cause) {
        super(message, cause);
    }
}

File: backend/src/crowntest/exceptions/ScreenshotFailedException.ts
export class ScreenshotFailedException extends Error {
  constructor(message: string, cause?: Error) {
    super(message);
    this.name = 'ScreenshotFailedException';
    if (cause) {
      this.stack += `\nCaused by: ${cause.stack}`;
    }
  }
}
